File: Sopra/finished_project/sopra-fs19-group03-client/src/components/shared/models/outgoingMessages/LobbyCredentials.js

class LobbyCredentials {
    constructor(token, mode){
        this.token = token;
        this.mode = mode;
    }
    setToken(token){this.token = token;}
    getToken(){return this.token;}
    setMode(mode){this.mode = mode;}
    getMode(){return this.mode;}
}
export default LobbyCredentials;

File: Sopra/finished_project/sopra-fs19-group03-server/src/main/java/ch/uzh/ifi/seal/soprafs19/dto/incoming/LobbyCredentials.java
package ch.uzh.ifi.seal.soprafs19.dto.incoming;


import java.io.Serializable;

public class LobbyCredentials implements Serializable, Message {
    private String token;
    private String mode;

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    public String getMode() {
        return mode;
    }

    public void setMode(String mode) {
        this.mode = mode;
    }
}